spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/README.adoc

```diff
@@ -43,22 +43,21 @@ do so with the following command:
 2.  Create, enable billing on a project on the
 https://console.cloud.google.com[Google Cloud Console].
 3.  Enable the
-https://console.cloud.google.com/flows/enableapi?apiid=runtimeconfig.googleapis.com[Runtime
+https://console.cloud.google.com/apis/api/runtimeconfig.googleapis.com/overview[Runtime
 Config API].
 4.  https://cloud.google.com/docs/authentication/getting-started#creating_the_service_account[Create a service account] and
 set the credentials using the `GOOGLE_APPLICATION_CREDENTIALS` environment variable or
-link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/README.adoc[using GCP Starter Core properties].
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/README.adoc[using GCP Starter Core properties]. (Note that when testing locally, the https://cloud.google.com/sdk/gcloud/reference/beta/auth/application-default/login[Cloud SDK application default credentials] are not supported by the Runtime Configuration API.)
```

Comment:
What's the warning trying to prevent/suggest?

Reply:
`org.springframework.web.client.HttpClientErrorException: 403 Forbidden`

Reply:
Should we tell users what they should do, instead of using app default credentials?
Is it just providing credentials in any other possible mean?

Reply:
Yes, we're telling them here to create a service account and specify it via `GOOGLE_APPLICATION_CREDENTIALS`.